ansible/roles/sortinghat/tasks/main.yml

```diff
@@ -16,13 +16,13 @@
 
 - name: Check databases list
   debug:
-    msg: "{{ databases }}"
+    msg: "{{ databases|unique }}"
```

Comment:
Because `unique` is used in many places, I think it would make more sense to set this filter on the task `Create a list of databases`, so it's only called once.